src/material-experimental/mdc-radio/radio.ts

```diff
@@ -50,6 +51,14 @@ export const MAT_RADIO_GROUP_CONTROL_VALUE_ACCESSOR: any = {
   multi: true
 };
 
+/**
+ * Injection token that can be used to inject instances of `MatRadioGroup`. It serves as
+ * alternative token to the actual `MatRadioGroup` class which could cause unnecessary
+ * retention of the class and its component metadata.
+ */
+export const MAT_RADIO_GROUP =
```

Comment:
FYI: This is a separate token in the MDC-based implementation as the non-MDC implementation should not interfere with the MDC-based one.